index.html

```diff
@@ -0,0 +1,637 @@
+            </div>
+            <div class="featured1">
+              <img src="images/3.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+            </div>
+
+            <div class="featured1">
+              <img src="images/4.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+            </div>
+
+            <div class="featured1">
+
+              <img src="images/5.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+
+            <div class="featured1">
+
+              <img src="images/7.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+
+            <div class="featured1">
+              <img src="images/9.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+          </div>
+          <div class="col-lg-4 col-md-6 top d-flex flex-column">
+            <h4 class="heading">TOP STORY</h4>
+            <img src="images/federal.png" alt="">
+            <p><a href="#">Federal Reserve Slashes Interest Rates to Near 0% Amid Coronavirus Pandemic</a></p>
+            <p>The central bank also urged banks to supply credit to the economy through the elimination of reserve requirements.</p>
+            <div class="culture">
+              <div class="culture1">
+                <img src="images/a.png" alt="">
+                <p class="top-p">Will I Get a Refund If My Trip Is Canceled Because of the Coronavirus?</p>
+                <p class="top-p1">Some insurance companies have been adjusting their policies in view of the outbreak, but travelers should check with their airlines, hotels and travel agents before filing a claim with insurers.</p>
+              </div>
```

Comment:
- [ ] You don't have the section's title (Culture and Travel)

- [ ]  The images in this section should have the same height.

- [ ] Check the font colors and weights of the titles and the descriptions to match the original page.